packages/core/src/item/itemStyle.js

```diff
@@ -0,0 +1,18 @@
+export const css = props => ({
```

Comment:
any reason not to make there exports default?

Reply:
I wanted each style file to have it's own name and the styles file to give it a more general name.
in the internals style file you will see "css", "wrapperCss" etc and in the central styles file you would have "treeCss", "inputWrapperCss"
avoiding each internal style file to consider all other files

Reply:
What? I don't understand the relation.
You have a default export - it doesn't mandate any name on how you import it.

Did I misunderstand?

Reply:
changed